async function fetchData(){

    try {
        let data = await Promise.reject('Failed');
    } catch (error){
        console.log('Error:', error);
    }
}

fetchData();